### PR TITLE
[Debezium] Implementing bits + other string types.

### PR DESCRIPTION
**Pull request by Tang8330**
Implementing the following:

* Bit (parsing as `[]byte` and returning b64 encoded string)
* Interval and XML string passthroughs

### CODE REVIEW

File: lib/debezium/types.go [hunk 1]
```diff
@@ -28,11 +28,16 @@ const (
...
+	JSON     SupportedDebeziumType = "io.debezium.data.Json"
+	Enum     SupportedDebeziumType = "io.debezium.data.Enum"
+	EnumSet  SupportedDebeziumType = "io.debezium.data.EnumSet"
+	UUID     SupportedDebeziumType = "io.debezium.data.Uuid"
+	LTree    SupportedDebeziumType = "io.debezium.data.Ltree"
+	Interval SupportedDebeziumType = "io.debezium.time.Interval"
+	XML      SupportedDebeziumType = "io.debezium.data.Xml"
+
+	// Bytes
+	Bit SupportedDebeziumType = "io.debezium.data.Bits"
```
Comment (nathan-artie) [inline lib/debezium/types.go:40]: Nit can we call this `Bits`? `Bit` makes me think this is just a single bit.

File: lib/debezium/converters/basic.go [hunk 2]
```diff
@@ -40,3 +41,19 @@ func (Int64Passthrough) Convert(value any) (any, error) {
 
 	return value, nil
 }
+
+type Bit struct{}
```
Comment (nathan-artie) [inline lib/debezium/converters/basic.go:45]: Could call this `Base64` since it's a converter that produces base64 encoded values.

**Review by nathan-artie — APPROVED** [review]

File: lib/debezium/types.go (hunk 1 above)
Comment (Tang8330) [reply lib/debezium/types.go:40]: Yeah sure

File: lib/debezium/converters/basic.go (hunk 2 above)
Comment (Tang8330) [reply lib/debezium/converters/basic.go:45]: sgtm

**Pull request merged**